var/spack/repos/builtin/packages/py-graphql-relay/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyGraphqlRelay(PythonPackage):
+    """Relay library for graphql-core."""
+
+    homepage = "https://github.com/graphql-python/graphql-relay-py"
+    pypi = "graphql-relay/graphql-relay-3.2.0.tar.gz"
```

Comment:
```suggestion
    pypi = "graphql-relay/graphql-relay-2.0.1.tar.gz"
```
Doesn't break anything, just helps with `spack url summary`

Reply:
It's okay with me. I first wrote this recipe for version 3.2.0 but version 2.0.1 was ultimately used. I fear that the following comments are due to leftovers from my first draft.